### PR TITLE
Add feature lists for each use case

### PR DESCRIPTION
**Pull request by jasonyavorsky**
This adds feature lists to each use case page via data and a generator. It also allows going the other direction (from the product team strategy to the use cases.)

### CODE REVIEW

File: data/features.yml
Comment (jasonyavorsky) [inline data/features.yml:13]:
```suggestion
    - none
```

File: schema/use_cases.schema.json
Comment (jasonyavorsky) [inline schema/use_cases.schema.json]:
```suggestion

```

File: schema/use_cases.schema.json
Comment (jasonyavorsky) [inline schema/use_cases.schema.json]:
```suggestion
  }

```

File: data/features.yml
Comment (Joelkw) [inline data/features.yml]:
```suggestion
    - code_health
    - fixing_security_vulnerabilities
    - incident_response
    - code_reuse
    - developer_onboarding
```

**Review by Joelkw — APPROVED** [review]

**Comment by jasonyavorsky** [comment]
@benvenker @lguychard @macraig I would have thought you'd have had some features related to use cases - can you confirm you've checked and don't before I merge?

File: data/features.yml
Comment (jasonyavorsky) [inline data/features.yml:367]: @Joelkw I saw you left a suggestion with this same content, was that to avoid these appearing due to the embargo?

File: data/features.yml
Comment (jasonyavorsky) [reply data/features.yml]: Resolving since this was applied in a commit

File: data/features.yml
Comment (Joelkw) [reply data/features.yml:367]: @jyavorska thanks for checking! No, I think this is so high level that it's fine, since the use cases don't tell you anything about the feature, what it does, how it works, etc. That was probably me missing the final merge button, thought I merged them in, sorry.

**Comment by lguychard** [comment]
@jyavorska added use cases for Search features.

**Comment by jasonyavorsky** [comment]
@dan-mckean there are some features in here owned by repo management & delivery; are there any that you'd want to associate with any of the use cases?

File: data/features.yml
Comment (dan-mckean) [inline data/features.yml:452]: We don't own that, does the Growth team maybe?

File: data/features.yml
Comment (dan-mckean) [inline data/features.yml]: This is actually Cloud SaaS who owns User auth - though I'm not sure exactly how to name them in this file to avoid breaking things.

**Review by dan-mckean — COMMENTED** [review]
@jyavorska I've added a couple of comments. But I don't think there's any use cases that we impact directly enough to warrant listing against the things for either Repo Mgmt or Delivery.

File: data/features.yml
Comment (jasonyavorsky) [reply data/features.yml]:
```suggestion
  product_team: cloud_saas
```

This should do the trick. We'll get an error message if I get it wrong.

File: data/features.yml
Comment (jasonyavorsky) [reply data/features.yml:452]: Ok, thanks!

File: data/features.yml
Comment (jasonyavorsky) [inline data/features.yml]:
```suggestion
  product_team: growth_and_integrations
```

**Pull request merged**
